src/ci/docker/dist-x86_64-linux/Dockerfile

```diff
@@ -84,7 +84,8 @@ ENV HOSTS=x86_64-unknown-linux-gnu
 ENV RUST_CONFIGURE_ARGS \
       --enable-full-tools \
       --enable-sanitizers \
-      --enable-profiler
+      --enable-profiler \
+      --enable-compiler-args
```

Comment:
s/args/docs/

I think?

Reply:
Oops, I think you're right, will change this.

Reply:
Fixed.